### ISSUE TITLE
As a FS admin I want to be sure that the images for the Tree Species are correct.

### ISSUE THREAD

**Issue by ASprinkle** [nciinc/fs-permit-platform#551]
Update tree species images from the https://plants.usda.gov/java/

(This came out of a discussion I had with a FS user when showing the website)

**Comment by lpendem** [comment]
@ASprinkle @MelissaBraxton None of the current images are from USDA, and most of the color & closeup images in the USDA site have copyright credits. Can we use those images in this site? `Pacific Silver Fir` tree profile for example: https://plants.usda.gov/core/profile?symbol=ABAM

**Comment by ASprinkle** [comment]
I have no idea how copyright credits/rules apply here. I punt to @MelissaBraxton  ?

**Comment by lpendem** [comment]
@MelissaBraxton I accidentally pushed the new species images from USDA with copyrights on them in our site. I will keep/revoke those images based on your feedback for the question above.

**Comment by MelissaBraxton** [comment]
That's fine @lpendem. Current thinking is to follow guidelines here: https://www.nrcs.usda.gov/wps/portal/nrcs/main/plantmaterials/technical/cp/photo/terms/ 
But we have an email out to the USDA web master for definitive guidance, and we can update based on what we hear.

**Comment by lpendem** [comment]
@MelissaBraxton @tram @JaneZC We have added the photographer name to each of the photo and  an additional line for USDA Credits. Can you provide feedback?
<img width="733" alt="screen shot 2018-03-12 at 5 26 35 pm" src="https://user-images.githubusercontent.com/7645088/37310525-92102342-261a-11e8-91ad-a469bae4051f.png">

**Comment by sadlerw** [comment]
While this approach isn't *exactly* what they ask you to please use, I think it works.

**Comment by ASprinkle** [comment]
I am concerned somewhat about the "cluttered" look that may occur if we have a credit below each photo.  Could we potentially add some italicized or "hint" or other type of text at the bottom of the page that summarizes all the references/photo credits on the page if we need to have them?
@bryanmcfadden I am curious, what do you have in your files as a summary of comments people have made about the tree species section of the page as it is now on staging?

**Comment by sadlerw** [comment]
I don't think we can summarize based on the usage restrictions from the plant db site.  I still find it somewhat strange that they allowed photographers to contribute images and retain copyright to them BUT...  how about this?  Bryan came up with it.

<img width="638" alt="screen shot 2018-03-14 at 9 02 35 am" src="https://user-images.githubusercontent.com/2058839/37406980-9f3625e2-2766-11e8-988a-c9c453c42e64.png">

**Comment by sadlerw** [comment]
Here is the final design - @ASprinkle @MelissaBraxton @tram 

<img width="745" alt="screen shot 2018-03-14 at 9 21 48 am" src="https://user-images.githubusercontent.com/2058839/37408009-36fd9dcc-2769-11e8-9475-f85e28c5c781.png">

**Comment by ASprinkle** [comment]
Thanks Will..  I am curious to see how it looks all together on one page. The way it is presented here is much cleaner - less cluttered than the first go.

**Comment by JaneZC** [comment]
I just want to add a thumbs up, but it isn't letting me do it. So, here's a written thumbs up to the new design. Its much better. :)

**Comment by MelissaBraxton** [comment]
Looking good!  There are a few images of odd size: Pacific Yew and Whitebark Pine. Is there any way we can make the size of those images consistent with the rest? Other than that, I say ship it@

**Comment by sadlerw** [comment]
If we limit the images to the same width and height we are not going to be able to use images from the USDA site that have a different aspect ratio than whatever size we pick.  I think maybe we should set a max-width of 140px so that they all line up the same width on the page but the heights can vary with the aspect ratio.  We should also make each one clickable to open up the image by itself so that people can see it in the original size like we are doing the maps.

**Comment by ASprinkle** [comment]
@spendem is the clickable tree image task completed?

**Comment by lpendem** [comment]
Yes @ASprinkle It's in staging. I just checked the task on the story.
